packages/docs/src/pages/en/getting-started/upgrade-guide.md

```diff
@@ -100,19 +119,28 @@ app.use(vuetify)
   - Allowed values are `'elevated'`, `'flat'`, `'tonal'`, `'outlined'`, `'text'`, or `'plain'`.
 - `text` prop has new purpose. It represents the text content of the alert, if default slot is not used.
 
-### v-btn
+### v-btn/v-btn-toggle
 
 - `active-class` prop has been renamed to `selected-class`.
-- `fab` is no longer supported.
+- `fab` is no longer supported. If you just need a round button, use `icon` prop or apply a `.rounded-circle` class
 - `flat` / `outlined` / `text` / `plain` props have been combined into a single `variant` prop.
 - `depressed` has been renamed to `variant="flat"`.
 - `retain-focus-on-click` has been removed, buttons use `:focus-visible` instead.
+- `v-btn-toggle` needs `mandatory="force"` prop to achieve the same behaviour as `mandatory` prop in v2
+- `v-btn` has a new size `x-large`, which is why you have to down-shift previously used sizes by one to get the same size as in v2. E.g. `small` becomes `x-small`
 
 ### v-checkbox/v-radio/v-switch
 
 - `input-value` prop has been renamed to `model-value`.
 - `on-icon` and `off-icon` props have been renamed to `true-icon` and `false-icon`.
 - `on-value` and `off-value` props have been renamed to `true-value` and `false-value`.
+- `v-switch` now uses an input of type "checkbox", it does no longer have an `aria-checked` attribute to assert in tests.
+- `v-switch` no longer has `aria-role="switch"`, use `aria-role="checkbox"` to match in tests
+- `v-checkbox`'s label slot should no longer contain a `<label>` as it is already wrapped with one
+
+### v-form
+
+- `validate()` now always returns a Promise that needs to be awaited. It also resolves on validation errors! Use `valid` and `errors` properties returned to determine form state
```

Comment:
```suggestion
- `validate()` now returns a [`Promise<FormValidationResult>`](/api/v-form/#exposed-validate) instead of a boolean. Await the promise then check `result.valid` to determine form state. 
```